x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_attributes/network/dns_top_domains.ts

```diff
@@ -84,6 +84,7 @@ export const getDnsTopDomainsLensAttributes: GetLensAttributes = (
                 params: {
                   query: 'PTR',
                 },
+                // @ts-expect-error upgrade typescript v4.9.5
                 indexRefName: 'filter-index-pattern-0',
```

Comment:
Hey @angorayc it looks like you edited this code ~10 months ago, maybe you'd like to look into this change.
There's no `indexRefName` in the `FilterMeta` type:

https://github.com/elastic/kibana/blob/bceeea80ee6ecaa8acdc2f323bba6c10b786f36e/packages/kbn-es-query/src/filters/build_filters/types.ts#L69-L85